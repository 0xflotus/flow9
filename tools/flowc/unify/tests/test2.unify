import maybe;

// Essence of something going wrong in test16
[None<α5>] c= α0;
[α7]  c=  α0;
Some<α6>  c=  α7;
finalize