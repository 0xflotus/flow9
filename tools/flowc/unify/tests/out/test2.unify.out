α0=[{None<α5> .. *}]
α7={Some<α6> .. *}
α100={None<α5> .. *}

Final:
α0=[Maybe<?>]
α5=?
α6=?
α7=Some<?>
α100=Maybe<?>
